src/main/java/org/opensearch/flowframework/model/WorkflowNode.java

```diff
@@ -50,7 +52,7 @@ public class WorkflowNode implements ToXContentObject {
     /** The field defining the timeout value for this node */
     public static final String NODE_TIMEOUT_FIELD = "node_timeout";
     /** The default timeout value if the template doesn't override it */
-    public static final String NODE_TIMEOUT_DEFAULT_VALUE = "15s";
+    public static final TimeValue NODE_TIMEOUT_DEFAULT_VALUE = new TimeValue(10, SECONDS);
```

Comment:
This is good for this PR.  I'm thinking I'd like to repurpose the `WORKFLOW_REQUEST_TIMEOUT` setting to be used here (and it's already a `TimeValue` too)... but we can handle that later.